test/readdlm.jl

```diff
@@ -162,7 +162,7 @@ let i18n_data = ["Origin (English)", "Name (English)", "Origin (Native)", "Name
         "Ireland", "Sinead O'Connor", "Éire", "Sinéad O'Connor",
         "Israel", "Yehoram Gaon", "ישראל", "יהורם גאון",
         "Italy", "Fabrizio DeAndre", "Italia", "Fabrizio De André",
-        "Japan", "KUBOTA Toshinobu", "日本", "久保田    利伸",
+        "Japan", "KUBOTA Toshinobu", "日本", "久保田    利伸",
```

Comment:
I really don't think that can happen. And if we want to test that, we should generate a file ourselves that cover all different white spaces....... (It's probably as likely (or less likely) to happen as tab or other random whitespace and none of those are tested....)